aviary/subsystems/propulsion/motor/motor_builder.py

```diff
@@ -54,18 +54,18 @@ def build_mission(self, num_nodes, aviary_inputs):
         return MotorMission(num_nodes=num_nodes, aviary_inputs=aviary_inputs)
 
     # def get_constraints(self):
-        # if self.include_constraints:
-        #     constraints = {
-        # Dynamic.Mission.Motor.TORQUE_CON: {
-        #     'upper': 0.0,
-        #     'type': 'path'
-        # }
-        # # TBD Gearbox torque constraint
-        #     }
-        # else:
-        #     constraints = {}
-
-        # return constraints
+    #     if self.include_constraints:
+    #         constraints = {
+    #     Dynamic.Mission.Motor.TORQUE_CON: {
+    #         'upper': 0.0,
+    #         'type': 'path'
+    #     }
+    #     TBD Gearbox torque constraint
+    #         }
+    #     else:
+    #         constraints = {}
+
+    #     return constraints
```

Comment:
remove?

Reply:
Since we're going to eventually need this code for the separate gearbox component I'm keeping this in until that happens